music21/chord/__init__.py

```diff
@@ -3698,7 +3698,7 @@ def sortDiatonicAscending(self, *, inPlace=False):
             returnObj = self
         else:
             returnObj = copy.deepcopy(self)
-        returnObj._notes.sort(key=lambda x: (x.pitch.diatonicNoteNum, x.pitch.ps))
+        sortDiatonicAscending(returnObj._notes)
```

Comment:
This request takes a documented and tested method and splits it into two functions, one of which has no documentation or tests.

Reply:
See below -- I still prefer this to be in the chord itself.